bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx109-en-features.html

```diff
@@ -0,0 +1,118 @@
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% from "macros-protocol.html" import split with context %}
+
+{% extends "firefox/whatsnew/base.html" %}
+
+{% block page_title %}{{ ftl('whatsnew-page-title') }}{% endblock %}
+{% block page_desc %}{{ ftl('whatsnew-page-description') }}{% endblock %}
+
+{% block page_css %}
+  {{ css_bundle('protocol-split') }}
+  {{ css_bundle('firefox_whatsnew') }}
+  {{ css_bundle('firefox_whatsnew_109_en_features') }}
+{% endblock %}
+
+{% set _entrypoint = 'mozilla.org-whatsnew109-features' %}
+{% set _utm_campaign = 'whatsnew109' %}
+
+{% block site_header %}{% endblock %}
+
+{% block wnp_content %}
+  <h1 class="mzp-c-section-heading">Firefox helps you stay private and productive</h1>
+
+  {% call split(
+    block_class='mzp-l-split-center-on-sm-md mzp-l-split-reversed',
+    body_class='mzp-l-split-v-center',
+    media_class='mzp-l-split-h-center',
+    image=resp_img('img/firefox/whatsnew/whatsnew109-en/shield.png',
+      srcset={ 'img/firefox/whatsnew/whatsnew109-en/shield-high-res.png': '2x' },
+      optional_attributes={ 'class': 'mzp-c-split-media-asset'}
+    ),
+  ) %}
+    <h2>See which trackers Firefox blocked for you this week</h2>
+    <p>We use Enhance Tracking Protection to automatically protect your privacy while you browse.</p>
```

Comment:
**Nit:** I used `18px` font-size for the rest of the WNP because the design seemed to want something a little larger than `16px`. Let's update it to `18px` to look more consistent?